apps/user-profile/src/app/pages/settings-page/settings-authorization/settings-authentication.component.html

```diff
@@ -17,10 +17,8 @@ <h1 class="page-subtitle mt-5">{{'AUTHENTICATION.MFA' | customTranslate | transl
 <!--  <mat-slide-toggle #toggle color="primary">{{'AUTHENTICATION.MFA_TOGGLE' | customTranslate | translate}}</mat-slide-toggle>-->
   <div [class.hide-table]="exporting" class="card mt-2">
     <div class="card-body">
-      <perun-web-apps-table-options (exportEnd)="exporting = false" (exportStart)="exporting = true"
-                                    [exporter]="exporter"></perun-web-apps-table-options>
       <div class="overflow-auto">
-        <table #exporter="matTableExporter" [dataSource]="dataSource" class="w-100" mat-table matTableExporter>
+        <table [dataSource]="dataSource" class="w-100" mat-table>
```

Comment:
You only removed old matTbaleExporter and you didn't replace it by new solution. If it was your purpose, then just ignore this comment.